edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/storage/MessageStore.cs

```diff
@@ -55,31 +55,31 @@ public void SetTimeToLive(TimeSpan timeSpan)
             Events.TtlUpdated(timeSpan);
         }
 
-        public async Task AddEndpoint(string endpointId)
+        public async Task AddEndpoint(string messageQueueId)
```

Comment:
Updated these based on what I saw its' usage really is from StoringEndpoint

Reply:
This change makes sense.. but - 
1. this is old code, and I would rather not change it at this point
2. we use endpointId for this everywhere else in the code (that was originally what we were calling it). 
So I would suggest we keep it as it is to minimize the changes here..

Reply:
Sounds good, I am okay with reverting back to endpointId for the variable as long as it is understood that it doesn't seem to always be the case (at least based on what I saw)

Reply:
So, maybe a comment will suffice?